### PR TITLE
ADD a SQL audit tool

### PR DESCRIPTION
**Pull request by sjjian**
SQLE is a SQL audit platform for DBA or developer. 
docs: https://github.com/actiontech/sqle/blob/main/README_en.md

### CODE REVIEW

**Review by shlomi-noach — COMMENTED** [review]
Looks OK - a couple nits:

- GitHub README has link to enterprise edition; not a deal breaker
- How do you feel about adding a clear link to the English README.md file inside the main README.md file?

**Review by dveeden — CHANGES_REQUESTED** [review]
I think we should link to the English README

File: README.md [hunk 1]
```diff
@@ -119,9 +119,9 @@ This list accepts and encourages pull requests. See [CONTRIBUTING](https://githu
 - [Liquibase](http://www.liquibase.org/) - Source control for your database
 - [Shift](https://github.com/square/shift) - An application that helps you run schema migrations on MySQL databases
 - [Skeema](https://www.skeema.io) - Declarative pure-SQL schema management system for MySQL and MariaDB, with support for sharding and external online schema change tools
+- [SQLE](https://github.com/actiontech/sqle) - SQLE is a SQL audit platform for DBA or developer
```
Comment (dveeden) [inline README.md:122]:
```suggestion
- [SQLE](https://github.com/actiontech/sqle/blob/main/README_en.md) - SQLE is a SQL audit platform for DBA or developer
```

**Review by dveeden — APPROVED** [review]

**Comment by shlomi-noach** [comment]
@dveeden feel free to merge PRs which you approve.

**Pull request merged**